src/Components/Buttons.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+
+
+//Creates a class of buttons. Allows use of props 
+class Button extends React.Component{
+
+
+
+//Renders the page. Creates buttons 
+    render(){
+        return ( 
+       <div>
+           <button className="button" id="startButton" disabled={this.props.startDisabled} onClick= {this.props.handleStart}> Start Game</button> {/* Create Start Button */}
+           <button disabled={true} className="button" disabled={this.props.guessDisabled} id= "guessButton" onClick= {this.props.handleGuess}> Guess</button> {/* Create Guess Button */}
+           <button disabled={true} className="button" disabled={this.props.quitDisabled} id= "quitButton" onClick= {this.props.handleQuit}> Quit</button> {/* Create Quit Button */}
```

Comment:
There are currently 2 `disabled=` properties on these buttons currently

Rather then passing props for each specific button being enabled or disabled, you could pass one of them and say `{this.props.startDisabled}` and `{!this.props.startDisabled}`
Doing this would cut back on the size of state as well as the amount of code written into each props being passed around

Reply:
Good advice, changing now